spec/test_authenticator.rb

```diff
@@ -0,0 +1,14 @@
+require 'minitest/pride'
+require 'minitest/autorun'
+
+require_relative '../lib/authenticator.rb'
+
+class TestAuthenticator < MiniTest::Test
+  def test_admin_has_correct_params
+    assert_equal(true, Authenticator.authenticate?('foo', 'bar'))
```

Comment:
it's better to use simply `assert Authenticator.authenticate?('foo', 'bar')`, it's equivalent